dev/InfoBar/TestUI/InfoBar.cs

```diff
@@ -4,7 +4,8 @@
 using Windows.UI.Xaml;
 using Windows.UI.Xaml.Controls;
 using Windows.Foundation;
-
+using InfoBar_TestUI;
```

Comment:
>InfoBar_TestUI [](start = 6, length = 14)

Delete this, the control shouldn't reference the test page

Reply:
Deleted w/ the latest push!